### ISSUE TITLE
Configurable behavior on "key not found"

### ISSUE THREAD

**Issue by yrtimiD** [doshprompt/angular-localization#50]
Please make "key not found" behavior configurable.
For example, I'd like to see the missing key instead of not very helpful %%KEY_NOT_FOUND%%

**Comment by doshprompt** [comment]
Yes I think that is a good idea. I have been toying with the idea myself, but in the meantime you may look in the console log to see the missing key.

**Comment by renatoch** [comment]
Another option for a quick fix is to wrap the getString function in another function, and handle the key_not_found message returning the desired string and whatever other behavior needed.
